fridge/utilities/mcnp_cell_writer.py

```diff
@@ -0,0 +1,174 @@
+                              inner, outer, universe, importance, comment):
+    """
+        Combinatorial geometry to create a cell which is outside the inner surface and inside the outer surface.
+
+        args:
+            fuel_assembly(class): holds information on the surface number to be used
+            material_id (int): ID number which tells what material to reference from in the data card
+            material_density (float): atom density of the material being used
+            inner (int): the surface number of the inner surface
+            outer (int): the surface number of the outer surface
+            universe (int): the universe number associate with this particular cell
+            importance (int): particle importance
+            comment (str): an accomanpying string to describe the RHP
+        return:
+            mcnp_output (str): string containing the input line for MCNP
+            mcnp_length_warning (bool): warns if the mcnp line is longer than 80 characters
+    """
+    mcnp_length_warning = False
+    mcnp_output = str(fuel_assembly.cell_number) + " " + str(material_id) + " " + str(round(material_density, 7)) + \
+                  "   " + str(inner) + " -" + str(outer) + "      u=" + str(universe) + " imp:n=" + str(importance) + \
+                  " $" + comment
+
+    if len(mcnp_output) > 80:
+        mcnp_length_warning = True
+        print("\033[1;37:33mWarning: Cell %d has a line that is longer than 80 characters", fuel_assembly.cell_number)
+    fuel_assembly.cell_number += 1
+    return mcnp_output, mcnp_length_warning
+
+
+def mcnp_make_cell(fuel_assembly, material_id, material_density, inner, universe, importance, comment):
+    """
+        Combinatorial geometry to create a cell on the inside of one surface
+
+        args:
+            fuel_assembly(class): holds information on the surface number to be used
+            material_id (int): ID number which tells what material to reference from in the data card
+            material_density (float): atom density of the material being used
+            inner (int): the surface number of the inner surface
+            universe (int): the universe number associate with this particular cell
+            importance (int): particle importance
+            comment (str): an accomanpying string to describe the RHP
+        return:
+            mcnp_output (str): string containing the input line for MCNP
+            mcnp_length_warning (bool): warns if the mcnp line is longer than 80 characters
+    """
+    mcnp_length_warning = False
+    mcnp_output = str(fuel_assembly.cell_number) + " " + str(material_id) + " " + str(np.round(material_density, 7)) + \
+                  "   -" + str(inner) + "      u=" + str(universe) + " imp:n=" + str(importance) + " $" + comment
+
+    if len(mcnp_output) > 80:
+        mcnp_length_warning = True
+        print("\033[1;37:33mWarning: Cell %d has a line that is longer than 80 characters", fuel_assembly.cell_number)
```

Comment:
You can do some built in stuff with the Exception library for warning. Instead of print('Warning Message'), you do warnings.warn('Warning Message'). Just a though.